lib/palettes.cpp

```diff
@@ -7703,16 +107,15 @@ void PaletteState::Init()
       MaxTextureSize = std::min(MaxTextureSize, 4096);
       {
          std::lock_guard<std::mutex> lk{init_mtx};
-         Init_Palettes();
```

Comment:
Remove the lock and mutex, it was here to protect the init, which we do not have anymore 🙌

Reply:
done 👍

Reply:
mutex still remains 😬 , so it cannot compile...